openmeter/ent/schema/billing.go

```diff
@@ -0,0 +1,418 @@
+package schema
```

Comment:
should we separate files per table?

Reply:
We usually do seperate file per service. That's why I haven't split it.